databases/week3/week3.sql

```diff
@@ -0,0 +1,129 @@
+DROP DATABASE IF EXISTS mealsharing;
+CREATE DATABASE mealsharing;
+USE mealsharing;
+DROP TABLE IF EXISTS reservation;
+CREATE TABLE `reservation` (
+  `id` int(10) NOT NULL PRIMARY KEY,
+  `number_of_guests` int(10)  NOT NULL,
+  `meal_id` int(10)  NOT NULL,
+  `created_date` date  NOT NULL,
+  `contact_phonenumber` varchar(255)  NOT NULL,
+  `contact_name` varchar(255)  NOT NULL,
+  `contact_email` varchar(255)  NOT NULL
+);
+DROP TABLE IF EXISTS meal;
+
+CREATE TABLE `meal` (
+  `id` int(10)  NOT NULL PRIMARY KEY,
+  `title` varchar(255)  NOT NULL,
+  `description` text  NOT NULL,
+  `location` varchar(255)  NOT NULL,
+  `when` datetime  NOT NULL,
+  `max_reservations` int(10)  NOT NULL,
+  `price` decimal  NOT NULL,
+  `created_date` date  NOT NULL
+);
+DROP TABLE IF EXISTS review;
+CREATE TABLE `review` (
+  `id` int(10)  NOT NULL PRIMARY KEY,
+  `title` varchar(255)  NOT NULL,
+  `description` text  NOT NULL,
+  `meal_id` int(10)  NOT NULL,
+  `stars` int(10)  NOT NULL,
+  `created_date` date NOT NULL
+);
+
+ ALTER TABLE `reservation` ADD FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`);
+
+ ALTER TABLE `review` ADD FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`);
+ -- meal table
+ insert into  meal(id, title, description, location, `when`, max_reservations, price, created_date) values (1, 'chicken pasta','spicy','copenhagen','2020-10-20 12:10:15', '5', '200','2020-10-20');
```

Comment:
Insufficient data, consider adding few more meals with variations to work with all the tasks.